modules/virt-installing-virt-operator.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-virt-web.adoc
+
+[id="virt-installing-virt-operator_{context}"]
+= Installing the {VirtProductName} Operator
+
+You can install the *{VirtProductName}* Operator from the {product-title} web console.
```

Comment:
Not sure if you do this consistently elsewhere in your docs, but I don't think that {VirtProductName} should be in bold here.

Reply:
Oh, that was a holdover from when the sentence said something else and was referring to a UI element -- another great catch :)